### ISSUE TITLE
COPY FROM returns a SQLActionException with no details

### ISSUE THREAD

**Issue by jeffnappi** [crate/crate#1727]
While attempting to re-import previously backed up data, I got this error. Would help if there was at least a descriptive error message.
```
cr> copy posts from 's3://(valid path...)/posts_*' with (compression='gzip', overwrite_duplicates=true);

SQLActionException[null]
```

**Comment by chaudum** [comment]
hi @jeffnappi 

looks like there is an Exception without a message thrown. We'll have to take a look at this.

additionally it seems like the verbose mode of crash is broken :(
I've opened an issue in the project repo: https://github.com/crate/crash/issues/55

you could try to use the REST API to get more information
https://crate.io/docs/en/latest/sql/rest.html?highlight=error_trace#error-handling

best, christian

**Comment by jeffnappi** [comment]
@chaudum thanks - I'll keep you posted if I encounter it again.

**Comment by chaudum** [comment]
ok, will close it then.

**Comment by chaudum** [comment]
@jeffnappi the crash bug has been fixed in 0.11.4 https://github.com/crate/crash/issues/55
